__tests__/app.test.js

```diff
@@ -199,6 +199,87 @@ describe('GET /api/articles/:article_id/comments ERRORS', () => {
 }
 )
 
+describe('POST /api/articles/:article_id/comments', () => {
+test('Should add a new comment for a valid article_id ', () => {
+
+    const newComment = {
+        username: 'butter_bridge',
+        body: 'Test Comment'
+    }
+    return request(app)
+            .post('/api/articles/1/comments')
+            .send(newComment)
+            .expect(200)
+            .then(response => {
+                expect(response.body.comment).toBeTruthy();
+                expect(response.body.comment.author).toBe(newComment.username);
+                expect(response.body.comment.body).toBe(newComment.body);
+     })
+});
+
+
+})
+
```

Comment:
happy and sad path tests for the same endpoint should be in the same describe block :)